client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx

```diff
@@ -56,10 +56,9 @@ import type { TranslateResult } from 'i18n-calypso';
 // This will make converting to TS less noisy. The order of components can be reorganized later
 /* eslint-disable @typescript-eslint/no-use-before-define */
 
-export default function WPCheckoutOrderSummary( {
+export function WPCheckoutOrderSummary( {
 	siteId,
 	onChangeSelection,
-	nextDomainIsFree = false,
```

Comment:
If you don't need `nextDomainIsFree` here (I agree, getting it directly from the cart seems better), you can remove it from the types of this component and the places that render it.